### DISCUSSION TITLE
Adding to the SDK on directus.logout to clean up the authentication token

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#19422] [Feature Requests]
### Summary

Hello,

So, the `AuthenticationStorage` https://github.com/directus/directus/blob/main/sdk/src/auth/types.ts#L12-L15 handle the set and get, but does not handle removal of the token. When logging out using the SDK, perhaps at least I feel, the expected would be to remove the token, so the AuthenticationStorage should accept remove and clean up on login out.

If this something we want, I can raise an MR for it if that is the desired feature.

### Motivation

Better flow of authentification with the SDK

### Detailed Design

Auth -> Log out -> Clean up any token on the browse

**Comment by br41nslug** [comment]
The removal/clean-up of tokens is done by setting the token to null https://github.com/directus/directus/blob/main/sdk/src/auth/composable.ts#L35-L37 (https://github.com/directus/directus/blob/main/sdk/src/auth/composable.ts#L161)

Please do update your feature request with more information because:
> Auth -> Log out -> Clean up any token on the browse

Is not a detailed enough description of the feature you'd like to see implemented.

**Reply by juancarlosjr97** [reply]
Okay, I will review it again because it did not work as expected on my end. If that is the case, I will provide minimal steps to reproduce the bug, and I will either return to this discussion or raise a bug ticket accordingly.

Thank you @br41nslug

**Reply by br41nslug** [reply]
If you're running into an actual bug then please do make an issue for it 😄

**Reply by juancarlosjr97** [reply]
Yes, sure I will if that case :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
